modules/imgcodecs/test/test_grfmt.cpp

```diff
@@ -337,6 +342,32 @@ TEST(Imgcodecs_Pam, read_write)
 }
 #endif
 
+#ifdef HAVE_IMGCODEC_PFM
+TEST(Imgcodecs_Pfm, read_write)
+{
+  string folder = string(cvtest::TS::ptr()->get_data_path()) + "readwrite/";
+  string filepath = folder + "lena.pfm";
+
+  cv::Mat img = cv::imread(filepath, IMREAD_UNCHANGED);
+  ASSERT_FALSE(img.empty());
+
+  std::vector<int> params;
+  string writefile = cv::tempfile(".pfm");
+  EXPECT_NO_THROW(cv::imwrite(writefile, img, params));
+  cv::Mat reread = cv::imread(writefile, IMREAD_UNCHANGED);
+
+  string writefile_no_param = cv::tempfile(".pfm");
+  EXPECT_NO_THROW(cv::imwrite(writefile_no_param, img));
+  cv::Mat reread_no_param = cv::imread(writefile_no_param, IMREAD_UNCHANGED);
+
+  EXPECT_EQ(0, cvtest::norm(reread, reread_no_param, NORM_INF));
+  EXPECT_EQ(0, cvtest::norm(img, reread, NORM_INF));
+
+  remove(writefile.c_str());
+  remove(writefile_no_param.c_str());
```

Comment:
EXPECT_EQ(0, remove(tmp_file.c_str()));

Reply:
Static analyzer tool coverity.com dislike if `remove()` call is added without check: https://scan.coverity.com/projects/opencv?tab=overview

Also this check allows to detect on some platforms if file handles are not properly closed and `remove()` returns errors. Or just misused/wrong file name.

`get_data_path()` is legacy code.